content/authors.json

```diff
@@ -1,4 +1,10 @@
 {
+  "tsc": {
+    "name": "Technical Steering Committee",
+    "summary": "The AdoptOpenJDK Technical Steering Committee (TSC) ",
+    "twitter": null,
```

Comment:
why not using the Adopt twitter account?

Reply:
done